mappings/com/mojang/realmsclient/dto/RealmsServer.mapping

```diff
@@ -37,9 +37,9 @@ CLASS net/minecraft/class_4877 com/mojang/realmsclient/dto/RealmsServer
 		ARG 1 slots
 	METHOD method_25062 getName ()Ljava/lang/String;
 	METHOD method_25063 parseSlots (Lcom/google/gson/JsonArray;)Ljava/util/Map;
-		ARG 0 jsonArray
+		ARG 0 json
 	METHOD method_25064 setDescription (Ljava/lang/String;)V
-		ARG 1 motd
+		ARG 1 description
```

Comment:
`motd` is like the parameter in 20w09a, but replacing it with better ones should be good